src/dashboard/src/media/js/misc.js

```diff
@@ -139,3 +139,23 @@ $(document).ready(
                 $(this).closest('.preview-help-text').children('.preview').show();
               });
 });
+
+// Return true if user navigated to page via browser back/forward button.
+// Return false if user navigated directly or if browser doesn't support Navigation Timing API.
+function navigatedFromBackOrForwardButton() {
+  if (window.performance) {
+    // Navigation Timing v2 spec
+    var navEntries = window.performance.getEntriesByType('navigation');
+    if (navEntries.length > 0 && navEntries[0].type === 'back_forward') {
```

Comment:
Oddly I couldn't find one! Here's the section in the spec: https://www.w3.org/TR/navigation-timing-2/#sec-performance-navigation-types. All of the MDN docs and other places I looked used the "back_forward" string rather than a constant as well.